starter-code/homework 07/main.js

```diff
@@ -0,0 +1,21 @@
+
+$(document).ready(function(){
+
+for (var i=99;i>=0;i--){
+var $newThingLi = $('<li>');
+var $list= $('#Slist'); 
+var num=i;
+if (i==1){
+$newThingLi.text("1 bottle of beer on the wall, 1 bottle of beer. Take one down and pass it around, no more bottles of beer on the wall");
```

Comment:
Here it's better if you replace `i` rather than `1`
```suggestion
$newThingLi.text( i + " bottle of beer on the wall, "+ i + " bottle of beer. Take one down and pass it around, no more bottles of beer on the wall");
```